test/test_image.py

```diff
@@ -48,3 +48,50 @@ def test_gradcheck(self):
         input = utils.tensor_to_gradcheck_var(input)  # to var
         assert gradcheck(image.gaussian_blur, (input, kernel_size, sigma,),
                          raise_exception=True)
+
+class TestNormalize:
+    def test_smoke(self):
+        mean = [0.5]
+        std = [0.1]
+        repr = 'Normalize(mean=[0.5], std=[0.1])'
+        assert str(image.Normalize(mean, std)) == repr
+
+    def test_normalize(self):
+
+        # prepare input data
+        data = torch.ones(1, 2, 2)
+        mean = torch.tensor([0.5])
+        std = torch.tensor([2.0])
+
+        # expected output
+        expected = torch.tensor([0.25]).repeat(1, 2, 2).view_as(data)
+
+        f = image.Normalize(mean, std)
+        assert_allclose(f(data), expected)
+
+    def test_batch_normalize(self):
+
+        # prepare input data
+        data = torch.ones(2, 3, 1, 1)
+        data += 2
+        mean = torch.tensor([0.5, 1.0, 2.0])
+        std = torch.tensor([2., 2., 2.])
+
+        # expected output
+        expected = torch.tensor([1.25, 1., 0.5]).repeat(2, 1, 1).view_as(data)
+
+        f = image.Normalize(mean, std)
+        assert_allclose(f(data), expected)
```

Comment:
I would add a couple of test more with small image patches (2x2 or 4x4, up to you) to verify that works with 2d arrays and verify the following:
- batch 1 and 1mean/std
- batch N and 1 mean/std
- batch N and N mean/std